### PR TITLE
Amam/update footer and translation text

### PR DESCRIPTION
**Pull request by cakasuma**
- update footer SVG
- fix non-ascii characters breaking translation

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -186,11 +186,11 @@ function processFooter(selected_language_name) {
...
+                        TEXT: 'Outside the EU, financial products are offered by Binary (SVG) Ltd, Hinds Building, Kingstown, St. Vincent and the Grenadines; Binary (V) Ltd, Govant Building, Port Vila, PO Box 1276, Vanuatu, regulated by the Vanuatu Financial Services Commission ([_1]view licence[_2]); Binary (BVI) Ltd, Kingston Chambers, P.O. Box 173, Road Town, Tortola, British Virgin Islands, regulated by the British Virgin Islands Financial Services Commission ([_3]licence no. SIBA/L/18/1114[_4]); and Binary (FX) Ltd., Lot No. F16, First Floor, Paragon Labuan, Jalan Tun Mustapha, 87000 Labuan, Malaysia, regulated by the Labuan Financial Services Authority to carry on a money-broking business ([_5]licence no. MB/18/0024[_6])'.i18n(),
```
Comment (ashkanx) [inline src/index.js]: @cakasuma remove `St. Vincent and the Grenadines`
Talk to Mahdi or Negar for further clarifications.

File: src/index.js [hunk 2]
```diff
@@ -186,11 +186,11 @@ function processFooter(selected_language_name) {
...
                         TAGS: ['<a href="https://www.vfsc.vu/wp-content/uploads/2015/12/List-of-Licensees-under-Dealers-in-Securities-Licensing-Act-CAP-70-18.11.2016.pdf" target="_blank" rel="noopener noreferrer">', '</a>', '<a href="https://www.binary.com/download/regulation/BVI_license.pdf" target="_blank" rel="noopener noreferrer">', '</a>', '<a href="https://www.binary.com/download/regulation/Labuan-license.pdf" target="_blank">', '</a>'],
                     },
                     P3: {
-                        TEXT: 'This website’s services are not made available in certain countries such as the USA, Canada, Costa Rica, Hong Kong, Japan, or to persons under age 18.'.i18n()
+                        TEXT: 'This website\'s services are not made available in certain countries such as the USA, Canada, St. Vincent and the Grenadines, Hong Kong, Japan, or to persons under age 18.'.i18n()
```
Comment (ashkanx) [inline src/index.js]: same here

File: src/index.js (hunk 1 above)
Comment (msamprz) [reply src/index.js]: @ashkanx we should only remove it from places that say we do not offer services to this country, right? and this doesn't specify that we do not offer services to `St. Vincent and the Grenadines`; we have not removed it from Binary Static either in my PR.

Comment (ashkanx) [reply src/index.js]: That's correct

**Pull request merged**